tbd.schema.json

```diff
@@ -122,6 +122,18 @@
           "title": "Convex (other than parapet/overhang) roof/wall edge PSI",
           "type": "number"
         },
+        "ceiling": {
+          "title": "Intermediate ceiling (not floor) edge PSI",
+          "type": "number"
+        },
+        "ceilingconcave": {
+          "title": "Concave intermediate ceiling (not floor) edge PSI",
+          "type": "number"
+        },
+        "ceilingconvex": {
+          "title": "Convex intermediate ceiling (not floor) edge PSI",
+          "type": "number"
+        },
```

Comment:
Needs additional tests, e.g.:

- how to customize "ceiling" PSI factors
- how to override "ceiling" PSI factors as e.g. "rimjoists"

TO DO ...